pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -539,6 +586,38 @@ func selectVictimsOnNode(
 	return victims, numViolatingVictim, true
 }
 
+// PreNominateCandidate does some preparation work before nominating the candidate winner:
+// - Evict the victim pods
+// - Reject the victim pods if they are in waitingPod map
+// - Clear the low-priority pods' nominatedNodeName status if needed
+func PreNominateCandidate(c Candidate, fh framework.FrameworkHandle, cs kubernetes.Interface, pod *v1.Pod) error {
```

Comment:
Perhaps `PrepareCandidate`?

Reply:
Given that the only thing we did after this function is nominating the candidate, I think "PreNominate" is more descriptive than "Prepare".

Reply:
There is no function or canonical phase named NominateCandidate, and so it might be confusing what the prefix "pre" mean here. I still think PrepareCandidate (which is really what we are doing here, changing the node state to accomodate the pod) is the better option and matches well with the previous phases: find, select and prepare.

Reply:
ok, "prepare" should be good enough.